client/js/components/AccountLoginForm/AccountLoginForm.js

```diff
@@ -0,0 +1,147 @@
+          })
+        }>
+        {login => (
+          <View style={styles.accountLoginContainer}>
+            <Form
+              onSubmit={async values => {
+                try {
+                  const data = await login({
+                    variables: {
+                      email: values.email,
+                      password: values.password,
+                    },
+
+                    // todo add navigate to home page.
+                  });
+                  console.log(data);
+                  if (data) {
+                    this.props.navigation.navigate('Home');
+                  }
+                } catch (e) {
+                  console.log(e);
+                  console.log('Didnt work');
+                }
+              }}
+              render={({handleSubmit}) => (
+                <View style={styles.formHolder}>
+                  {/* start of email field */}
+                  <View style={styles.textField}>
+                    <Field
+                      name="email"
+                      style={styles.textField}
+                      render={({input, meta}) => (
+                        <TextInput
+                          style={{fontSize: 16, width: '60%'}}
+                          id="email"
+                          placeholder="User Email"
+                          placeholderTextColor="black"
+                          type="text"
+                          inputProps={{
+                            autoComplete: 'off',
+                          }}
+                          {...input}
+                        />
+                      )}
+                    />
+                    <Image
+                      style={{width: 20, height: 20, resizeMode: 'contain'}}
+                      source={EmailIcon}
+                    />
+                  </View>
+                  {/* End of email field */}
+
+                  {/* Start of password field */}
+                  <View style={styles.textField}>
+                    <Field
+                      name="password"
+                      render={({input, meta}) => (
+                        <TextInput
+                          id="password"
+                          style={{fontSize: 16, width: '60%'}}
+                          selectionColor="black"
+                          placeholder="Password"
+                          secureTextEntry={true}
+                          placeholderTextColor="black"
+                          inputProps={{
+                            autoComplete: 'off',
+                          }}
+                          {...input}
+                          type="password"
+                        />
+                      )}
+                    />
+                    <Image
+                      style={{width: 20, height: 20, resizeMode: 'contain'}}
+                      source={PasswordIcon}
+                    />
+                  </View>
+                  {/* end of password field */}
+
```

Comment:
move all the inline styles to styles.js file.

Reply:
Done